Source/diablo.cpp

```diff
@@ -238,6 +238,9 @@ BOOL diablo_get_not_running()
 {
 	SetLastError(0);
 	CreateEvent(NULL, FALSE, FALSE, "DiabloEvent");
+#ifdef HELLFIRE
+	CreateEventA(0, 0, 0, "HellfireEvent");

```

Comment:
```diff
- 	CreateEventA(0, 0, 0, "HellfireEvent");
+ 	CreateEventA(NULL, FALSE, FALSE, "HellfireEvent");
```

Reply:
Aw sad i missed that :(

Reply:
Also, `CreateEvent` in Diablo and `CreateEventA` in Hellfire, but perhaps that's how it should be?

Reply:
yep, fixed https://github.com/diasurgical/devilution/commit/c46a39b9fc736a31d02a243a0c4d76e663e669fd